src/lua/call.c

```diff
@@ -617,7 +621,9 @@ static int read_cb(lua_State*L)
   FD_ZERO(&fdset);
   FD_SET(myfileno, &fdset);
   dt_lua_unlock();
+  #if !defined(_WIN32)
```

Comment:
Admittedly I have never worked with lua before. Is this `select ()` an async I/O waiting for the fd to be available for reading a file? For async I/O I believe we can find a solution. Or when the `read_cb() `is called?